x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -81,21 +80,18 @@ export const CreateEngineButton: React.FC<CreateEngineButtonProps> = ({ disabled
     >
       <EuiPopoverTitle>
         <FormattedMessage
-          id="xpack.enterpriseSearch.content.searchApplications.createEngineDisabledPopover.title"
-          defaultMessage="Platinum only feature"
+          id="xpack.enterpriseSearch.content.searchApplications.createEngineTechnicalPreviewPopover.title"
```

Comment:
run `node scripts/i18n_check --fix` for updating removed i18n messages if they have been translated already.